README.md

```diff
@@ -141,6 +141,7 @@ If you are reading this it looks like you are looking to add an egg to your serv
     * [TyphoonLimbo](/minecraft/proxy/java/typhoonlimbo)
     * [Velocity](/minecraft/proxy/java/velocity)
     * [Waterfall](/minecraft/proxy/java/waterfall)
+    * [VIAaaS](/minecraft/proxy/java/viaaas)
```

Comment:
Please keep the alphabetical order by putting this above waterfall

Reply:
Ah, ok. Didn't think about that.